### PR TITLE
Add instructions for SSO configuration using SAML with Okta

### PR DESCRIPTION
**Pull request by timroster**
Example shows how to provide the required fields for a SAML 2.0 association with an IDP. Okta prefers to work with just the necessary fields rather than imports / exports of xml for the SAML association. It does provide and export xml, but this is not the preferred pattern for CP4D 3.5

### CODE REVIEW

File: 3.5/SSO_README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+# Configure SSO with SAML using Okta
+
+## Overview
+
+Cloud Pak for Data can be configured to use external authenitcation through a SAML provider. If this is to be implemented, it should be implemented immediately after installation of the base and before the addition of any end-users to CPD. These steps are based on the [documentation for single sign-on](https://www.ibm.com/support/knowledgecenter/SSQNUZ_3.5.0/cpd/install/saml-sso.html).
```
Comment (annumberhocker) [inline 3.5/SSO_README.md]: "authenitcation" misspelled

File: 3.5/SSO_README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+* Add an additional attribute needs to be added to specify the user identity. Presently the `nameID` property in the Okta SAML assertion is not parsed, requiring this additional attribute. In the **Attribute Statements (optional)** section add an attribute with label `username`, and specify the preferred format. For the value, select either `user.email` if the email is to be used, or if a simple string like the left-hand-side of the Okta login should be used, type in an expression like: `String.substringBefore(user.login, "@")`
```
Comment (annumberhocker) [inline 3.5/SSO_README.md]: Extra word "Add" at beginning of sentence

File: 3.5/SSO_README.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+    ![custom attribute](./images/saml_additional_attribute.png)
+
+Click on the **Next** button to continue towards the page with information needed to add to the CP4D instance saml configuration file. To get to the Setup instructions page, let Okta know that this is an internal application and that it is not needed for Okta to reach out to the service provider for more formal support. After clearing the application information panel, click on **View Setup Instructions**
```
Comment (annumberhocker) [inline 3.5/SSO_README.md:41]: Inconsistent with CPD and CP4D references.  Probably should be one or the other.

File: 3.5/SSO_README.md (hunk 1 above)
Comment (timroster) [reply 3.5/SSO_README.md]: fixed in https://github.com/ibm-hcbt/cp4d-install/pull/3/commits/81e270188e020b8dc1b641ee6017216a4f63c953

File: 3.5/SSO_README.md (hunk 2 above)
Comment (timroster) [reply 3.5/SSO_README.md]: fixed in https://github.com/ibm-hcbt/cp4d-install/pull/3/commits/81e270188e020b8dc1b641ee6017216a4f63c953

File: 3.5/SSO_README.md (hunk 3 above)
Comment (timroster) [reply 3.5/SSO_README.md:41]: bulk replace to CP4D in https://github.com/ibm-hcbt/cp4d-install/pull/3/commits/81e270188e020b8dc1b641ee6017216a4f63c953

**Review by annumberhocker — APPROVED** [review]

**Review by annumberhocker — APPROVED** [review]
lgtm

**Pull request merged**